src/core/data.jl

```diff
@@ -1,23 +1,242 @@
+function count_repairable_items(network::Dict{String, Any})
+    if _IMs.ismultinetwork(network)
+        Memento.warn(_PMs._LOGGER, "count_repairable_items supports single networks.  Attempting to select network 0.")
+        if haskey(network["nw"],"0")
+            Memento.info(_PMs._LOGGER, "Network 0 found.")
+            nw = network["nw"]["0"]
+        else
+            Memento.error(_PMs._LOGGER, "Network 0 not found.")
+        end
+    else
+        nw = network
+    end
+
+    repairable_count = 0
+    for (comp_name, status_key) in _PMs.pm_component_status
+        for (i, comp) in get(nw, comp_name, Dict())
+            if haskey(comp, status_key) && comp[status_key] != _PMs.pm_component_status_inactive[comp_name] && haskey(comp, "damaged") && comp["damaged"] == 1
+                repairable_count += 1
+            end
+        end
+    end
+    
+    return repairable_count
+end
+
+
+""
+function get_repairable_items(network::Dict{String, Any})
+    ## TODO verify functionality
```

Comment:
relevant comment?  In any case, a test should exist that checks this function.

Reply:
no longer relevant, will be removed